aggregation/src/main/java/fk/prof/aggregation/model/FinalizedAggregationWindow.java

```diff
@@ -59,7 +64,7 @@ protected Header buildHeaderProto(int version, WorkType workType) {
     return Header.newBuilder()
         .setFormatVersion(version)
         .setWorkType(workType)
-        .setAggregationEndTime(endWithTolerance.atOffset(ZoneOffset.UTC).format(DateTimeFormatter.ISO_ZONED_DATE_TIME))
+        .setAggregationEndTime(endedAt == null ? null : endedAt.atOffset(ZoneOffset.UTC).format(DateTimeFormatter.ISO_ZONED_DATE_TIME))
```

Comment:
No, I meant both builder and DTO, but looks like neither is possible. The API is extremely stupid, it seems, read: https://groups.google.com/forum/#!topic/protobuf/b0gS4wpjuIo and https://groups.google.com/forum/#!topic/protobuf/No9bBRh3Wp0 

It seems they wanted to support some "optimizations" by being GC unfriendly, which they didn't get right either. Dumbness rules!